build.gradle

```diff
@@ -117,6 +117,17 @@ subprojects { subproject ->
 							version = springBootVersion
 						}
 					}
+					else if (jacksonProjects.contains(subproject.name)) {
+						parent {
```

Comment:
I somehow think that `parent` is still wrong direction.
Looks like we need to resolve an actual version for every BOM-imported dependency.
See how it is done in SF: https://github.com/spring-projects/spring-framework/blob/master/gradle/publish-maven.gradle#L21, although the rely on the `id "io.spring.dependency-management" version "1.0.7.RELEASE" apply false`, which is not standard Gradle feature...